pkg/spectrumx/spectrumx.go

```diff
@@ -225,6 +226,7 @@ func parametersApplied(device *v1alpha1.NicDevice, params []types.ConfigurationP
 	return true, nil
 }
 
+// RuntimeConfigApplied checks if the desired Spectrum-X runtime spec is applied to the device
```

Comment:
Consider changing name to be more self explanatory  
```suggestion
// IsRuntimeConfigApplied checks if the desired Spectrum-X runtime spec is applied to the device
```

Reply:
Let's change it in a separate PR. This one is pretty big already